src/client/sound/sound_data.cpp

```diff
@@ -22,6 +22,8 @@ with this program; ifnot, write to the Free Software Foundation, Inc.,
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
+#include <algorithm>
+
 #include "sound_data.h"
 
 #include "sound_constants.h"
```

Comment:
Just some reordering:
```suggestion
#include "sound_data.h"

#include "sound_constants.h"
#include <algorithm>
```

Reply:
do we even have a defined include order? I could have sworn system headers go first.

Reply:
For what I could see its a bit all over the place. But now I notice most have system last.

https://github.com/minetest/minetest/blob/master/src/client/camera.cpp # Alphabetic
https://github.com/minetest/minetest/blob/master/src/client/clientmedia.h # System last

Reply:
AFAIK, we usually include the .cpp's header first (`"sound_data.h"`). Other than that, we don't have any consistent order.
(My personal preference is: first .cpp's header, then own headers, then library headers, then system headers. This way you'll most easily notice when an include is missing in a header.)